packages/riipen-ui/src/components/Chip.jsx

```diff
@@ -30,21 +40,41 @@ class Chip extends React.Component {
      */
     disabled: PropTypes.bool,
 
+    /**
+     * Whether to invert colors on hover
+     */
+    hover: PropTypes.bool,
```

Comment:
This props is slightly confusing to me. If I mouse over an element and it changes color, my brain tells me it should be clickable or interactable (apparently that isn't a real world) in some way. This doesn't seem to be the case here though. It looks like from the code it's just an aesthetic thing? This seems weird to me, but it's probably more of a Maribel question at this point. Can you verify that my thought process here is correct?

Reply:
removed as it wasnt even being used.